internal/manager/config.go

```diff
@@ -152,6 +153,7 @@ func (c *Config) FlagSet() *pflag.FlagSet {
 	flagSet.StringVar(&c.KongCustomEntitiesSecret, "kong-custom-entities-secret", "", `A Secret containing custom entities for DB-less mode, in "namespace/name" format`)
 
 	// Kubernetes configurations
+	flagSet.StringVar(&c.GatewayAPIControllerName, "gateway-api-controller-name", "", "The controller name to match on Gateway API resources.")
```

Comment:
Since we have an effective default value under the hood, we should list it here:

```suggestion
	flagSet.StringVar(&c.GatewayAPIControllerName, "gateway-api-controller-name", "konghq.com/kic-gateway-controller", "The controller name to match on Gateway API resources.")
```